staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -1715,3 +1727,111 @@ func TestWaitUntilWatchCacheFreshAndForceAllEvents(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, forceAllEvents, "the target method should instruct the caller to ask for all events in the cache (full state)")
 }
+
+type fakeStorage struct {
+	pods []example.Pod
+	storage.Interface
+}
+
+func newObjectStorage(fakePods []example.Pod) *fakeStorage {
+	return &fakeStorage{
+		pods: fakePods,
+	}
+}
+
+func (m fakeStorage) GetList(ctx context.Context, key string, opts storage.ListOptions, listObj runtime.Object) error {
+	podList := listObj.(*example.PodList)
+	podList.ListMeta = metav1.ListMeta{ResourceVersion: "12345"}
+	podList.Items = m.pods
+	return nil
+}
+func (m fakeStorage) Watch(_ context.Context, _ string, _ storage.ListOptions) (watch.Interface, error) {
+	return newDummyWatch(), nil
+}
+
+func BenchmarkCacher_GetList(b *testing.B) {
+	testCases := []struct {
+		name            string
+		totalObjectNum  int
+		expectObjectNum int
+	}{
+		{
+			name:            "totalObjectNum=5000, expectObjectNum=50",
```

Comment:
nit: you could construct this name via fmt.Sprintf when calling t.Run, and then there would be zero chance it wouldn't match the test case settings.

Reply:
Thank you for your suggestion. already fixed.